src/Microsoft.VisualStudio.Extensibility.Testing.Xunit.Shared/Harness/VisualStudioInstanceFactory.cs

```diff
@@ -11,15 +11,19 @@ namespace Xunit.Harness
     using System.Linq;
     using System.Reflection;
     using System.Text;
+    using System.Threading;
     using System.Threading.Tasks;
     using Microsoft.VisualStudio.Setup.Configuration;
+    using Microsoft.VisualStudio.Threading;
     using Microsoft.Win32;
     using DTE = EnvDTE.DTE;
     using File = System.IO.File;
     using Path = System.IO.Path;
 
     internal sealed class VisualStudioInstanceFactory : MarshalByRefObject, IDisposable
     {
+        private const int ReportTimeMinute = 5;
```

Comment:
It would be better to put this in a config file.
😀 But I don't familiar with this repo, would be good if someone could point me in the right direction.

Reply:
It should be fine here. If this ever happens, it means Visual Studio did something wrong and not the extension.